src/Bellhop.js

```diff
@@ -115,7 +115,9 @@ export class Bellhop extends BellhopEventDispatcher {
       // Check to see if the data was sent as a stringified json
       if ('string' === typeof data) {
         try {
+          if (data && !data.startsWith('webpack')) {
```

Comment:
For now I'm going to request that this additional check be removed. See comment for more info